src/app/CMakeLists.txt

```diff
@@ -32,6 +32,26 @@ endif()
 # -----------------------------------------------------------------------------
 add_executable(qbt_app)
 
+# Generate lproj folders for the translations
+set(LPROJ_FOLDERS "")
+foreach(TS_FILE IN LISTS QBT_TS_FILES)
+    string(FIND "${TS_FILE}" "_" POS)
+    math(EXPR START "${POS} + 1")
+    string(SUBSTRING "${TS_FILE}" ${START} -1 LPROJ_FOLDER)
+    string(REPLACE ".ts" ".lproj" LPROJ_FOLDER "${LPROJ_FOLDER}")
+    # @ is not valid as a language code for a lproj folder on MacOS
+    string(REPLACE "@" "-" LPROJ_FOLDER "${LPROJ_FOLDER}")
+    list(APPEND LPROJ_FOLDERS "${LPROJ_FOLDER}")
+endforeach()
+foreach(LPROJ_FOLDER IN LISTS LPROJ_FOLDERS)
```

Comment:
So maybe incorporate all the jobs in the `foreach` above? Is there any strong reason to perform two loops?